pennylane/init.py

```diff
@@ -183,321 +131,643 @@ def random_layers_normal(n_layers, n_wires, n_rots=None, mean=0, std=0.1, seed=N
 
     Keyword Args:
-        n_rots (int): number of rotations, if ``None``, ``n_rots=n_wires``
-        low (float): minimum value of rotation angles
-        high (float): maximum value of rotation angles
         seed (int): seed used in sampling the parameters, makes function call deterministic
 
     Returns:
-        list of parameter array
+        list of parameter arrays
     """
     if seed is not None:
         np.random.seed(seed)
 
-    if n_rots is None:
-        n_rots = n_wires
+    kwargs = {'n_layers': n_layers, 'n_wires': n_wires, 'seed': seed}
 
-    params = np.random.uniform(low=low, high=high, size=(n_rots,))
-    return [params]
+    theta_1 = cvqnn_layers_theta_uniform(**kwargs)
+    phi_1 = cvqnn_layers_phi_uniform(**kwargs)
+    varphi_1 = cvqnn_layers_varphi_uniform(**kwargs)
+    r = cvqnn_layers_r_normal(**kwargs)
+    phi_r = cvqnn_layers_phi_r_uniform(**kwargs)
+    theta_2 = cvqnn_layers_theta_uniform(**kwargs)
+    phi_2 = cvqnn_layers_phi_uniform(**kwargs)
+    varphi_2 = cvqnn_layers_varphi_uniform(**kwargs)
+    a = cvqnn_layers_a_normal(**kwargs)
+    phi_a = cvqnn_layers_phi_a_uniform(**kwargs)
+    k = cvqnn_layers_kappa_normal(**kwargs)
 
+    return [theta_1, phi_1, varphi_1, r, phi_r, theta_2, phi_2, varphi_2, a, phi_a, k]
 
-def random_layer_normal(n_wires, n_rots=None, mean=0, std=0.1, seed=None):
-    r"""Creates a list of a single parameter array for :func:`~.RandomLayer`, drawn from a normal distribution.
 
-    The number of parameter array is ``(n_rots,)`` and each parameter is drawn
-    from a normal distribution with mean ``mean`` and standard deviation ``std``.
-    The parameters define the rotation angles of the randomly positioned rotations applied in each layer.
+def cvqnn_layers_theta_uniform(n_layers, n_wires, low=0, high=2*pi, seed=None):
+    r"""Creates a parameter array for the ``theta`` input to the interferometers of :func:`~.CVNeuralNetLayers`.
 
-    Args:
-        n_wires (int): number of qubits
+        The parameters are drawn from a uniform distribution.
 
-    Keyword Args:
-        n_rots (int): number of rotations, if ``None``, ``n_rots=n_wires``
-        mean (float): mean of parameters
-        std (float): standard deviation of parameters
-        seed (int): seed used in sampling the parameters, makes function call deterministic
+        The shape of the arrays is ``(n_layers, n_wires*(n_wires-1)/2)``.
 
-    Returns:
-        list of parameter array
-    """
+        Args:
+            n_layers (int): number of layers of the CV Neural Net
+            n_wires (int): number of modes of the CV Neural Net
+
+        Keyword Args:
+            low (float): minimum value of uniform distribution
+            high (float): maximum value of uniform distribution
+            seed (int): seed used in sampling the parameters, makes function call deterministic
+
+        Returns:
+            array: parameter array
+        """
     if seed is not None:
```

Comment:
*almost* feels like we should make a simple helper function do avoid a lot of duplicate code. Something like
```
def _sample_uniform(low, high, size, seed=None):
    if seed is not None:
        np.random.seed(seed)
    return np.random.uniform(low=low, high=high, size=size)
```

Reply:
(more maintainable too, since absolutely every `init` function would be fed through the same 2 helper functions)